packages/react-server-cli/src/startServer.js

```diff
@@ -313,6 +313,7 @@ const logProductionWarnings = ({hot, minify, jsUrl, longTermCaching}) => {
 		if (process.env.NODE_ENV !== "production") { //eslint-disable-line no-process-env
 			logger.warning("-- NODE_ENV is not set to \"production\".");
 		}
+		logger.info(`NODE_ENV is set to ${process.env.NODE_ENV}`);
```

Comment:
Does it make sense to keep the `not production` warning above if we're adding this?